routing/car_directions.cpp

```diff
@@ -101,6 +101,12 @@ size_t CarDirectionsEngine::GetTurnDirection(IRoutingResult const & result, size
                                              NumMwmIds const & numMwmIds,
                                              RoutingSettings const & vehicleSettings, TurnItem & turnItem)
 {
+  // This is for jump from initial point to start of the route.
+  /// @todo Sometimes results of GetPossibleTurns are empty, sometimes are invalid.
```

Comment:
What should be done to fix it?

Reply:
The best will be to fix GetPossibleTurns()

Reply:
Comment added.